### ISSUE TITLE
Courses Of Action view broken under Malwares > Knowledge > Attack Patterns

### ISSUE THREAD

**Issue by tmlamb** [LuatixHQ/opencti#1949]
## Description

The Courses of Action view under Malware Knowledge in the 5.2.1 release seems to be broken. I've recreated the issue on a fresh local build as well as the demo on https://demo.opencti.io.

## Environment

1. OS (where OpenCTI server runs): N/A (demo.opencti.io)
3. OpenCTI version: 5.2.1
4. OpenCTI client: frontend
5. Other environment details:

## Reproducible Steps

1. Navigate to **Knowledge > Arsenal** and select a Malware
1. Navigate to **Knowledge > Attack Patterns**
1. Select **Courses of Action view** (path is `/dashboard/arsenal/malwares/<malware_id>/knowledge/attack_patterns?searchTerm=&viewMode=courses-of-action`)

## Expected Output

Courses of Action viewType should display correctly.

## Actual Output

Courses of Action view fails to load, with the following error displayed:

> Error
> An unknown error occurred. Please contact your administrator or the OpenCTI maintainers.

And the browser console has the following error:

> TypeError: n is undefined

**Comment by tmlamb** [comment]
I did a little more testing and I think I narrowed the issue down to the CSV export button:

https://github.com/OpenCTI-Platform/opencti/blob/91734ba98ce9305c597a81d67a0f44b2bf3d8fb6/opencti-platform/opencti-front/src/components/ExportButtons.js#L252
```javascript
{csvData && (
  <Tooltip title={t("Export to CSV")}>
    <CSVLink data={csvData}>
      <IconButton aria-haspopup="true" color="primary" size="large">
        <FileDelimitedOutline />
      </IconButton>
    </CSVLink>
  </Tooltip>
)}
```

Removing the MUI Tooltip component that wraps the react-csv CSVLink component seems to resolve the issue, so maybe there is a problem with compatibility between these, perhaps introduced when either react-csv or MUI versions were updated recently.

In other words, this seems to work:

```javascript
{csvData && (
  <CSVLink data={csvData}>
    <IconButton aria-haspopup="true" color="primary" size="large">
      <FileDelimitedOutline />
    </IconButton>
  </CSVLink>
)}
```

Edit: I think the MUI documentation for [Tooltip](https://mui.com/components/tooltips/#custom-child-element) and [component wrapping](https://mui.com/guides/composition/#wrapping-components) in general supports the theory that this is an issue with compatibility between this react-csv component and the MUI Tooltip.
